CodeStarterCamp_Week3/main.swift

```diff
@@ -9,36 +9,76 @@
 import Foundation
 
 class Person {
-    var money: Int = 0
+    var name: String
+    var money: Int
     
-    func buyCoffee() {
-        
+    init(name: String, money: Int) {
+        self.name = name
+        self.money = money
+    }
+    
+    func buyCoffee(coffeeShop: CoffeeShop, nameOfCoffee: Coffee) {
```

Comment:
요 메서드에선 전달인자 레이블을 사용해봐도 좋을 것 같아요!
메서드 네이밍이 참 어렵죠 ..? ㅎㅎ  👀
```suggestion
    func buy(coffee: Coffee, at coffeeShop: CoffeeShop) {
```